build.sbt

```diff
@@ -356,7 +356,7 @@ lazy val rosetta = (project in file("modules/rosetta"))
 lazy val dagL1 = (project in file("modules/dag-l1"))
   .enablePlugins(AshScriptPlugin)
   .enablePlugins(JavaAppPackaging)
-  .dependsOn(kernel, shared % "compile->compile;test->test", nodeShared, testShared % Test)
+  .dependsOn(kernel, shared % "compile->compile;test->test", nodeShared % "compile->compile;test->test", testShared % Test)
```

Comment:
do we still need it? we already have a `testShared` module so there should be no point to share `test` phase of `nodeShared` 🤔